cora/signal/lss.py

```diff
@@ -247,10 +247,14 @@ class CalculateMultiFrequencyAngularPowerSpectrum(task.SingleTask):
     xromb : int, optional
         Gauss-Legendre quadrature order for integrating C_ell over radial
         bins. (Used Romberg integration in a previous version, hence the
-        name xromb.) xromb=0 turns off this integral. Default: 2.
+        name xromb.) xromb=0 turns off this integral. Default: 2. When
+        dealing with nonlinear matter power spectrum, for sub-percent
+        accuracy up to l ~ 1500, xromb = 3 is recommended.
```

Comment:
Just by convention, the default value should be at the end of the argument description.
```suggestion
        name xromb.) xromb=0 turns off this integral. When
        dealing with nonlinear matter power spectrum, for sub-percent
        accuracy up to l ~ 1500, xromb = 3 is recommended. Default: 2. 
```